vocabularies/UI.xml

```diff
@@ -1360,6 +1360,18 @@ The trend is
         </Member>
       </EnumType>
 
+      <EnumType Name="ValueListRangeOperator">
```

Comment:
Why is this in UI if it is only used in Common?

Reply:
Initially placed it here, as it felt like a subset of `SelectionRangeOptionType` above to me. Moved it to `Common` (and renamed to `ValueListParameterOperator`) with 7ff504d7eecf8e1236e872df202ab8ed4d2f4a6f.